websites/J/JAVX/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "Minx",
+    "id": "829346671544107088"
+  },
+  "service": "JAVX",
+  "description": {
+    "en": "JAVX - Watch JAV, Watch Porn, Watch Sex Video 18+ Full HD Uncensored",
+    "vi_VN": "JAVX - Xem JAV, Xem Porn, Xem Phim Sex 18+ Full HD Không Che Mới Nhất"
+  },
+  "url": "javx.net",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/0Jwz0Ct.png",
+  "thumbnail": "https://i.imgur.com/Llt7QOj.png",
```

Comment:
I meant like, change these blurs that you applied to something like idk black bar or something, just to don't see even that it is porn.

Reply:
Oh got it, I'll fix it right away.